prov/efa/src/efa_user_info.c

```diff
@@ -572,19 +572,11 @@ int efa_getinfo(uint32_t version, const char *node,
 	struct fi_info *dgram_info_list, *rdm_info_list;
 	int err;
 
-	if (hints && hints->ep_attr && hints->ep_attr->type == FI_EP_DGRAM) {
-		err = efa_user_info_get_dgram(version, node, service, flags, hints, info);
-		if (err)
-			return err;
-		return 0;
-	}
+	if (hints && hints->ep_attr && hints->ep_attr->type == FI_EP_DGRAM)
+		return efa_user_info_get_dgram(version, node, service, flags, hints, info);
```

Comment:
This change should be in the 1st commit, but went into 2nd commit.

Reply:
I see, thank you! Fixed.